content/docs/integrations/github-discovery/index.md

```diff
@@ -10,7 +10,7 @@ integrationType: Integration
 
```

Comment:
If you add a `lastUpdated` key to the front matter YAML of this page, it will tell the user when the document was last updated, which is useful if they are trying to determine whether or not they should trust it.